test/widgets/stream-boss.ts

```diff
@@ -26,6 +26,7 @@ test('Set stream-boss health', async t => {
   t.pass();
 });
 
+// TODO: flaky
```

Comment:
Should this comment be down with the skipped test in this file?  I'm a little confused.